src/cli.rs

```diff
@@ -305,7 +305,7 @@ mod cli_tests {
     where
         T: Debug + AsRef<OsStr>,
     {
-        let params = CompileParameters::parse(&args);
+        let params = CompileParameters::parse(args);
```

Comment:
is this a merge? strange

Reply:
Yes, i changed the compile parameters to a &[] instead of Vec in a previous commit, not sure why this shows up here